aclogview/CM_Movement.cs

```diff
@@ -664,6 +669,40 @@ public class Jump_NonAutonomous : Message {
         }
     }
 
+    public class LifestoneMaterialize : Message
+    {
+        public uint ObjectId;
+        public int unknown1;
+        public Position Position;
+        public int unknown2;
+        public int unknown3;
+
+        public static LifestoneMaterialize read(BinaryReader binaryReader)
+        {
+            LifestoneMaterialize newObj = new LifestoneMaterialize();
+            newObj.ObjectId = binaryReader.ReadUInt32();
+            newObj.unknown1 = binaryReader.ReadInt32();
+            newObj.Position = Position.read(binaryReader);
+            newObj.unknown2 = binaryReader.ReadInt32();
+            newObj.unknown3 = binaryReader.ReadInt32();
```

Comment:
Perfect, I was also getting quite confused as to how that would work. Thank you for the further clarification, it should now be set!